framework/db/sqlite/QueryBuilder.php

```diff
@@ -134,13 +134,13 @@ public function resetSequence($tableName, $value = null)
                 $key = reset($table->primaryKey);
                 $tableName = $db->quoteTableName($tableName);
                 $value = $this->db->useMaster(function (Connection $db) use ($key, $tableName) {
-                    return $db->createCommand("SELECT MAX('$key') FROM $tableName")->queryScalar();
+                    return $db->createCommand("SELECT MAX($key) FROM $tableName")->queryScalar();
                 });
             } else {
                 $value = (int) $value - 1;
             }
             try {
-                $db->createCommand("UPDATE sqlite_sequence SET seq='$value' WHERE name='{$table->name}'")->execute();
+                return "UPDATE sqlite_sequence SET seq='$value' WHERE name='{$table->name}'";
             } catch (Exception $e) {
```

Comment:
this try and catch does not make sense anymore if the command is not executed here.